python/tvm/autotvm/graph_tuner/dynamic_programming_tuner.py

```diff
@@ -88,6 +89,17 @@ def _backward(self):
         for key, val in self._out_nodes_dict.items():
             if not val:
                 output_idx_list.append(key)
+
+        # Restrict number of output nodes to avoid numpy reshape error
+        if len(output_idx_list) > MAX_OUTPUT_NODES:
+            msg = "The number of outputs in graph is larger than upper " \
+                  "limit: %s vs %s. Usually this is caused by too many " \
+                  "LAYOUT_FIXED_OP in graph. Switch to greedily select schedule." \
```

Comment:
It's not very clear how user can take action. does graph-tuner switch to greedy mode automatically or user needs to do something else?

Reply:
This is for noticing purpose only. User doesn't need to do anything.

Reply:
maybe add "No action required at this moment. We will continuously improve graph tuner"